### PR TITLE
Craig 65

### PR DESCRIPTION
**Pull request by jvallexm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "icse-react-assets",
-  "homepage": "http://ibm.github.io/icse-react-assets",
-  "version": "0.1.4",
+  "homepage": "http://pages.github.ibm.com/icse/icse-react-assets",
```
Comment (isaaceichelberger) [inline package.json:3]: homepage should be `http://ibm.github.io/icse-react-assets`

File: storybook/src/stories/Forms/EncryptionKey.stories.js [hunk 2]
```diff
@@ -17,7 +17,19 @@ export default {
       control: "none",
       type: { required: true }, // required prop or not
     },
-    ["data.interval_month"]: {
+    ["data.endpoints"]: {
```
Comment (isaaceichelberger) [inline storybook/src/stories/Forms/EncryptionKey.stories.js:20]: endpoint

File: src/components/forms/EncryptionKeyForm.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import {
   buildFormFunctions,
 } from "../component-utils";
 import { IcseFormGroup } from "../Utils";
-import { IcseNameInput, IcseToggle } from "../Inputs";
+import { IcseNameInput, IcseToggle, IcseTextInput } from "../Inputs";
```
Comment (isaaceichelberger) [inline src/components/forms/EncryptionKeyForm.js:7]: do we need to use the new endpoint dropdown in this form somewhere?

File: src/components/forms/EncryptionKeyForm.js [hunk 4]
```diff
@@ -126,21 +153,26 @@ class EncryptionKeyForm extends Component {
 EncryptionKeyForm.defaultProps = {
   data: {
     name: "",
-    interval_month: 12,
+    rotation: 12,
     root_key: false,
     force_delete: false,
+    dual_auth_delete: false,
+    key_ring: "",
```
Comment (isaaceichelberger) [inline src/components/forms/EncryptionKeyForm.js:160]: add endpoint

**Review by isaaceichelberger — CHANGES_REQUESTED** [review]

File: src/components/forms/EncryptionKeyForm.js (hunk 3 above)
Comment (jvallexm) [reply src/components/forms/EncryptionKeyForm.js:7]: yup

**Review by isaaceichelberger — APPROVED** [review]

**Review by isaaceichelberger — APPROVED** [review]

**Review by hjweds — APPROVED** [review]

**Pull request merged**